pkg/operation/botanist/component/etcd/monitoring.go

```diff
@@ -189,6 +190,19 @@ const (
     annotations:
       description: Etcd data restoration was triggered, but has failed.
       summary: Etcd data restoration failure.
+
+  # etcd backup failure alert
+  - alert: KubeEtcdBackupRestore{{ .Role }}Down
```

Comment:
I only see the alert `KubeEtcdBackupRestoreMainDown` but in my Prometheus targets I see:
`kube-etcd3-backup-restore-events` and `kube-etcd3-backup-restore-main`. I would expect to see 2 Alerts as well. Or is this expected?

Reply:
`etcd-events` is configured with backup disabled. Hence, you may not see alerts for it.